core/src/main/java/org/keycloak/crypto/integration/CryptoIntegration.java

```diff
@@ -0,0 +1,47 @@
+package org.keycloak.crypto.integration;
+
+import org.jboss.logging.Logger;
+import org.keycloak.common.util.BouncyIntegration;
+import org.keycloak.common.util.reflections.Reflections;
+
+/**
+ * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
+ */
+public class CryptoIntegration {
```

Comment:
Should probably consider getting rid of BouncyIntegration and merging that stuff into here. Can we done as a follow-up perhaps?